internal/provider/resource_access_jaas_group.go

```diff
@@ -39,10 +41,14 @@ type groupInfo struct{}
 func (j groupInfo) Info(ctx context.Context, getter Getter, diag *diag.Diagnostics) (genericJAASAccessData, names.Tag) {
 	groupAccess := jaasAccessModelResourceGroup{}
 	diag.Append(getter.Get(ctx, &groupAccess)...)
+	// set roles to empty set because we don't support assign roles to group
+	emptyRoleSet, diagEmptySet := types.SetValue(types.StringType, nil)
+	diag.Append(diagEmptySet...)
```

Comment:
A different approach is needed if the jaas access group doesn't have roles. If a user does put roles into the jaas access group resource, when you write the data back, terraform cli will error as the values are not the same.  I'm surprised testing hasn't revealed a problem here, though the tests haven't been updated for this resource.